tests/test_ida_features.py

```diff
@@ -1,24 +1,26 @@
 # run this script from within IDA with ./tests/data/mimikatz.exe open

```

Comment:
its easiest to just view the complete new file, rather than trying to parse this diff.